edx_proctoring/static/proctoring/js/views/proctored_exam_info.js

```diff
@@ -0,0 +1,184 @@
+edx = edx || {};
+
+(function(Backbone, $) {
+    'use strict';
+
+    edx.courseware = edx.courseware || {};
+    edx.courseware.proctored_exam = edx.courseware.proctored_exam || {};
+
+    var viewHelper, examStatusReadableFormat;
+    examStatusReadableFormat = {
+        created: {
+            status: gettext('Created'),
+            message: gettext('Your onboarding attempt has been created, but it has not yet been submitted.')
+        },
+        download_software_clicked: {
+            status: gettext('Download Software Clicked'),
+            message: gettext(
+                'You have downloaded the proctoring software, ' +
+                'but your onboarding exam has not yet been submitted.'
+            )
+        },
+        ready_to_start: {
+            status: gettext('Ready to start'),
+            message: gettext('You are ready to start your onboarding exam, but it has not yet been submitted.')
+        },
+        started: {
+            status: gettext('Started'),
+            message: gettext('You have started your onboarding exam, but it has not yet been submitted.')},
+        ready_to_submit: {
+            status: gettext('Ready to submit'),
+            message: gettext('You are ready to submit your exam')
+        },
+        second_review_required: {
+            status: gettext('Second Review Required'),
+            message: gettext('Your onboarding exam has been submitted, but it requires a second review.')
+        },
+        submitted: {
+            status: gettext('Submitted'),
+            message: gettext('Your onboarding exam has been submitted.')
+        },
+        verified: {
+            status: gettext('Verified'),
+            message: gettext('You can now take proctored exams in this course.')
+        },
+        rejected: {
+            status: gettext('Rejected'),
+            message: gettext('Your onboarding exam has been rejected. Please retry onboarding.')
+        },
+        error: {
+            status: gettext('Error'),
+            message: gettext('An error has occurred during your onboarding exam. Please retry onboarding.')
+        }
+    };
+
+    viewHelper = {
+        getExamAttemptStatus: function(status) {
+            if (status in examStatusReadableFormat) {
+                return examStatusReadableFormat[status].status;
+            } else {
+                return status;
+            }
+        },
+        getExamAttemptMessage: function(status) {
+            if (status in examStatusReadableFormat) {
+                return examStatusReadableFormat[status].message;
+            } else {
+                // what should be the default message?
```

Comment:
Not sure what a default message should be - would appreciate any thoughts on this!

Reply:
maybe just a status of 'Not Started'. I think the general reminder text ('You must complete the onboarding process prior..') is probably sufficient as a message.